tests/json-fixtures/test_transactions.py

```diff
@@ -5,6 +5,7 @@
 import rlp
 
 from eth.exceptions import UnrecognizedTransactionType
+from eth.vm.forks.shanghai.transactions import ShanghaiTransactionBuilder
```

Comment:
This is sort of useless and so was the `Merge` addition here... there are no transaction tests for Shanghai nor Merge